src/pp_metrics.erl

```diff
@@ -86,6 +124,9 @@ handle_cast({location, PubKeyBin}, #state{seen = SeenMap} = State) ->
     ok = inc_location(Location),
     NewSeenMap = maps:put(PubKeyBin, Location, SeenMap),
     {noreply, State#state{seen = NewSeenMap}};
+handle_cast({lns_metric, Address, MessageType, Status}, State) ->
+    ok = inc_lns_metric(Address, MessageType, Status),
```

Comment:
If this is a public named table, why not just increment it here and avoid the message passing?

Reply:
Sorry, in the calling process, I mean.

Reply:
Good thinking. inc moved to calling process, thanks.